TheGrove/woocommerce-gateway-converge
# Fixing PHPCS Errors (security-related)

Closes #9, Closes #19

As per the [assessment](https://10upwoo.wordpress.com/2024/02/07/woocommerce-extension-assessment-woocommerce-elavon-converge-eu-gateway/#complexity-maintainability), our strategy focuses solely on addressing security-related PHPCS errors and warnings. The list of errors/warnings is compiled [here](https://gist.github.com/iamdharmesh/972a5eaa577bc913b2beb834102762b3) (thanks @iamdharmesh), which aligns with our primary objective for this PR.

### Testing instructions

* Confirm all functionality works as expected, especially the payment process and the AJAX actions.
* Please check all backend/frontend pages and confirm no plain HTML tags are found. For example: please see the below screenshot

![image](https://github.com/TheGrove/woocommerce-gateway-converge/assets/25176325/adc0ac2c-2274-43a5-9145-49d15d4d9599)

* This PR also adds the workflow file for PHPCS so please make sure it runs successfully and passes.
